modules/snooker/src/main/ts/ephox/snooker/api/CopySelected.ts

```diff
@@ -153,7 +153,8 @@ const extract = (table: SugarElement<HTMLTableElement>, selectedSelector: string
   const unselectedCells = LayerSelector.filterFirstLayer(replica, 'th,td', (cell) => Selectors.is(cell, selector));
   Arr.each(unselectedCells, Remove.remove);
 
-  fillInGaps(list, replicaHouse, replicaStats, isSelected);
+  const rows = Arr.filter(list, (row) => row.section !== 'colgroup');
```

Comment:
Don't forget this needs a snooker changelog as well.

Reply:
Also would this filtering be better off in the `fillInGaps` function instead of being here? Imagine if someone else ends up calling the `fillInGaps` function with the full details list later, then they'll just run into this same issue.

- [ ] Consider moving the filtering logic into the fillInGaps function

Reply:
Have added a changelog and moved the filtering to `fillInGaps`